helix-term/src/ui/editor.rs

```diff
@@ -382,9 +387,16 @@ impl EditorView {
                                 if (pos.col as u16) < viewport.width + view.first_col as u16
                                     && pos.col >= view.first_col
                                 {
-                                    let style = Style::default()
-                                        .add_modifier(Modifier::REVERSED)
-                                        .add_modifier(Modifier::DIM);
```

Comment:
I would leave this part as is, since it's the match bracket (not inactive, we don't render cursors in unfocused docs at all)

Reply:
Making it themable through something like `ui.match_brace` would be nice (setting it to underline would be less visually jarring).

Reply:
Is now themable as `ui.cursor.match`